core/src/main/java/org/wildfly/extras/creaper/core/online/HttpModelControllerClient.java

```diff
@@ -181,26 +181,35 @@ private String getManagementRealm(String url) throws IOException {
 
         HttpPost post = new HttpPost(url);
         post.addHeader(HttpHeaders.CONTENT_TYPE, ContentType.APPLICATION_JSON.getMimeType());
+        // this is not necessary, but without this server produce parse error exception when authentication is disabled
+        // which may confuse user
+        post.setEntity(new StringEntity("[]"));
 
         CloseableHttpResponse response = null;
+        String content;
         try {
             response = defaultHttpClient.execute(post);
+            content = EntityUtils.toString(response.getEntity());
             response.close();
         } finally {
             defaultHttpClient.close();
```

Comment:
I asked the exact same question in the original PR, where the answer clearly was "no". But there's a change to this piece of code, so maybe this should be reevaluated.